exercise_05/isAdditionGreaterThanFifty.js

```diff
@@ -4,3 +4,18 @@
  * input: number (summand2)
  * output: boolean
  */
+/**
+ * 
+ * @param {number} numbersum1 - the first input 
+ * @param {number} numbersum2 - the second input
+ * @returns boolean - 
+ */
+
+function isAdditionGreaterThanFifty(numbersum1, numbersum2){
+    const  resultsum1 = (numbersum1 + numbersum2);
+    const resultsum2 =  resultsum1 >= 50;
```

Comment:
Avoid magic numbers. Use descriptive constants
```suggestion
    const resultsum2 =  resultsum1 >= THRESHOLD;
```